osu.Game/Scoring/ScoreImporter.cs

```diff
@@ -62,6 +62,11 @@ public ScoreImporter(RulesetStore rulesets, Func<BeatmapManager> beatmaps, Stora
                     api.Queue(req);
                     return null;
                 }
+                catch (Exception)
+                {
+                    Logger.Log($@"Score '{archive.Name}' failed to import: failed to parse replay headers.", LoggingTarget.Database);
+                    return null;
+                }
```

Comment:
I can't find myself agreeing with this. This eats the inner exception, and there is no way to ever recover it anymore. It will not be in logs, it will not be on sentry. Moreover it is feasible that the error message will be straight up counterproductive, because a random `IOException` due to lack of permissions or no space left on disk will get converted to "failed to parse replay headers", which I guess is technically _not false_, but also in this context has zero meaning anymore.

Reply:
I got halfway through adding exception output back , and for the 100th time I got stuck on `Logger.Error` being unusable because it's displaying notifications and sending to sentry.

Exception content definitely needs to be output in logs somehow.

Reply:
Have adjusted this accordingly.

Reply:
Still not going to be logged to sentry but having it in file logs is at least something I guess...